integration-provisioning/src/main/java/org/symphonyoss/integration/provisioning/service/CompanyCertificateService.java

```diff
@@ -78,17 +80,21 @@ public void init() {
    * @param application
    */
   public void importCertificate(Application application) {
-    LOGGER.info("Import company certificate: {}", application.getComponent());
-
     String pem = getPem(application);
-    CompanyCert companyCert = buildCompanyCertificate(application, pem);
 
-    String sessionToken = authenticationProxy.getSessionToken(DEFAULT_USER_ID);
+    if (StringUtils.isBlank(pem)) {
+      LOGGER.info("Skipping certificate importing for: {}", application.getComponent());
```

Comment:
I would rather keep this as is. What we want to cover here is the on-prem case where there won't be a .pem file available and hence we will not import it.